### DISCUSSION TITLE
Clarification requested regarding Act of Communication, Intentional Process

### DISCUSSION THREAD

**Discussion by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#404] [Announcements]
Suppose I have a camera set up in the forest to monitor flash floods, triggered by a motion sensor and a sound sensor.  When the motion sensor and sound sensor sense activity above a threshold,  a short video is recorded and sent to a server for later analysis. Is that sending of the video an Act of Communication? 

The definition of Intentional Action, superclass of Act of Communication is
>An Act in which at least one Agent plays a causative role and which is prescribed by some Directive Information Content Entity held by at least one of the Agents.

Some questions that arise:
1. Is the camera system an agent, i.e. capable of intentional acts?
2. If so, it would seem that there are many Artifacts that are in fact Agents.
3. If not, the closest agent to the event are people such as someone who helped design the camera system, someone who helped make it, someone who placed it in the forest, someone who turned it on, someone who ordered it placed. Are any of these realizing a causative role that would satisfy the definition. While these people play a causative role in some sense,  none of their acts are specific to a particular instance of the process described. Nonetheless is it intended that such agents satisfy the Intentional Act criterion "at least one Agent plays a causative role"?
4. If the answer to (3) is yes then 
    * Are there any communications that are not intentional processes? More generally, are all realizations of Artifact Functions also Acts?
    * Would all Mechanical Processes be Acts as they involved an Artifact, which involved an Agent that designed the Artifact? 
    * Should the domain of has accomplice, has agent, has accessory, and the subproperties of prescribed by all be Act?
    * Similarly would any realization of a role that 'has subordinate role' some role be an Act?
    * Would has accomplice, has accessory, has sender, has recipient all be subproperties of has agent?

There are more questions that arise. Basically, if the answer to (3) is yes then one would look for any mentions of Agent or Act, trace back to a related process, and ask whether that process must be an Act, and review relations whose domain is process and range (explicit or implied by definition) is an agent and ask if they are subproperties of has agent. 

I'm assuming that, although unstated, the domain of has agent is Act. Would that be correct?

Depending on some of the answers to these questions, quick fixes could result in asserted polyhierarchy, suggesting some refactoring be done in order to avoid that.

**Comment by nklsbckmnn** [comment]
1. I personally don't think so.
3. I think the causative role has to played within the process, not upstream of it to be an Act. #79 

So I don't think this is an Act of Communication at all.

**Comment by alanruttenberg** [comment]
In #79 I comment on the "causally active" criterion, which I think is difficult to evaluate. 

@eliasweatherfield I wasn't clear on which question you your (1) answers. Also, which "this" are you referring to in the last sentence?

**Comment by nklsbckmnn** [comment]
Sorry. 1 refers to 1. I don't think the camera system is an agent. "This" refers to the sending of the video.

**Comment by nklsbckmnn** [comment]
The case would be more complicated if the camera system were an advanced type of AI having intentions similar to humans or animals because the definition does not seem to exclude artifactual agents.
